Draggable prop for card

samiyaakhtar opened the pull request:
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

Added a draggable prop to card to allow non draggable cards like Recommendation card have normal cursor and other css
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/5572)

micahgodbolt commented on packages/dashboard/src/components/Card/Card.types.ts:
```diff
@@ -122,6 +122,11 @@ export interface ICardProps {
   callOnDidMount?: VoidFunction;
+
+  /**
+   * Whether the card is draggable or not
```
what is the default value?

micahgodbolt commented on packages/dashboard/src/components/Card/Card.tsx:
```diff
@@ -33,6 +33,7 @@ export class Card extends React.Component<ICardProps, ICardState> {
           cardTitle={cardFrameContent.cardTitle}
           cardDropDownOptions={cardFrameContent.cardDropDownOptions}
           href={cardFrameContent.href}
+          draggable={draggable === undefined ? true : draggable}
```
I'm guessing this means it's true by default? Typically we try to set props to be the exception case, so disableDrag is typically false, but can be set to try via <Foo disableDrag />

https://github.com/OfficeDev/office-ui-fabric-react/wiki/Component-Design#naming-guidance

samiyaakhtar commented:
Closing this one due to other package merge conflicts, and created a new one. @micahgodbolt, addressed your feedback in https://github.com/OfficeDev/office-ui-fabric-react/pull/5641
